Kitodo-DataManagement/src/main/java/org/kitodo/data/database/beans/Property.java

```diff
@@ -197,32 +173,65 @@ public void setValueList(List<String> valueList) {
         this.valueList = valueList;
     }
 
-    @Override
     public Integer getContainer() {
         if (this.container == null) {
             return 0;
         }
         return this.container;
     }
 
-    @Override
     public void setContainer(Integer order) {
         if (order == null) {
             order = 0;
         }
         this.container = order;
     }
 
-    public Process getProcess() {
-        return this.process;
+    public List<Process> getProcesses() {
+        if (this.processes == null) {
+            this.processes = new ArrayList<>();
+        }
+        return this.processes;
+    }
+
+    public void setProcesses(List<Process> processes) {
+        this.processes = processes;
+    }
+
+    public List<Template> getTemplates() {
+        if (this.templates == null) {
+            this.templates = new ArrayList<>();
+        }
+        return this.templates;
+    }
+
+    public void setTemplates(List<Template> templates) {
+        this.templates = templates;
+    }
+
+    public List<User> getUsers() {
+        if (this.users == null) {
+            this.users = new ArrayList<>();
+        }
+        return this.users;
+    }
+
+    public void setUsers(List<User> users) {
+        this.users = users;
+    }
+
+    public List<Workpiece> getWorkpieces() {
+        if (this.workpieces == null) {
+            this.workpieces = new ArrayList<>();
+        }
+        return this.workpieces;
     }
 
-    public void setProcess(Process process) {
-        this.process = process;
+    public void setWorkpieces(List<Workpiece> workpieces) {
+        this.workpieces = workpieces;
     }
 
-    @Override
-    public int compareTo(ProcessProperty o) {
-        return this.getTitle().toLowerCase().compareTo(o.getTitle().toLowerCase());
+    public int compareTo(Property property) {
+        return this.getTitle().toLowerCase().compareTo(property.getTitle().toLowerCase());
```

Comment:
I think it's not enough to compare the titles, you should also consider the actual value.